src/Features/CSharp/Portable/Ambiguity/CSharpAmbiguousTypeCodeFixProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.AmbiguityCodeFixProvider;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Ambiguity
```

Comment:
nit: namespaces here should match the feature name.  So this would be AliasAmbiguousType

Reply:
The idea was that all ambiguity code fixer would go into this namespace. Other candidates would be CS0121 for extension methods (could offer full qualification with transformation in a call to a static method) or CS0433 for same class in same namespace in different assemblies and maybe others.

Reply:
That's generally not how we organize the codebase though :-/